app/controllers/articles_controller.rb

```diff
@@ -3,7 +3,50 @@
 class ArticlesController < ApplicationController
   def index
     @articles = Article.accepted
-                       .search_by_content(params[:search]).order_list(params[:sort_by])
-                       .paginate(page: params[:page]).includes([:user])
+                  .search_by_content(params[:search]).order_list(params[:sort_by])
+                  .paginate(page: params[:page]).includes([:user])
+  end
+
+  def show
+    @article = Article.find(params[:id])
+  end
+
+  def edit
+    @article = Article.find(params[:id])
```

Comment:
@sheva233 please change `Article.find(params[:id])` to `current_user.articles.find`